data-archive/archive-jobs/mosip_credential/mosip_archive_job_credential.sh

```diff
@@ -0,0 +1,9 @@
+### -- ---------------------------------------------------------------------------------------------------------
+### -- Script Name		: Credential Archive Job
+### -- Deploy Module 		: Credentail DB (IDREPO)
+### -- Purpose    		: To Archive credential DB tables which are marked for archive.       
+### -- Create By   		: Kamesh Shekhar Prasad
+### -- Created Date		: Aug-2022
+
```

Comment:
please keep seperate .sh file for each python script.. keep similar suffix for them.

Reply:
done.